off_nitrogens/perturb_angle.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python
+
+#=============================================================================================
+# MODULE DOCSTRING
+#=============================================================================================
+
+"""
+perturb_angle.py
+
+Perturb geometry around an improper dihedral angle. The options are:
+ 1. Change valence angle without changing improper
+ 2. Change improper angle without changing valence angles
+
+For use in generating geometries to parameterize valence and improper angles
+
+By: Victoria Lim and Jessica Maat
+
+"""
+
+#=============================================================================================
+# GLOBAL IMPORTS
+#=============================================================================================
+
+import numpy as np
+import math
+import sys
+
+from calc_improper import *
```

Comment:
@bannanc, I wrote a test for `perturb_angle.py` in the file `test_perturb.py`. The Travis test is [failing](https://travis-ci.org/MobleyLab/off_nitrogens/jobs/374667080#L1931) though, because of this line. Is there a way I can both import it for normal use as well as import it for Travis use?

Reply:
Oh, sorry I just saw this, if you import it from your package name it should work so `from off_nitrogens import calc_improper as ci` or something similar, the `import *` is really not an ideal way to import because it over writes your name space.

Reply:
If I try this on my computer though, it doesn't have `off_nitrogens` as a module. Do I need to install it as a module? I'd like to have the script be called without requiring the module if that's possible.

Reply:
I think you could do it with a try/except statement? 
```
try:
    from off_nitrogens.calc_improper import *
except:
    from calc_improper import *
```
I would do it in that order. 

Although I'm not clear on why you wouldn't want to just install the module. You can use
`pip install -e .` and it will use a symbolic link to your python environment so you don't have to re-install when you change the script. 
OR 
you could always test it with this locally and switch it to `from off_nitrogens...` for the github account?